kms/api/CloudKmsTest/CloudKmsTest.cs

```diff
@@ -24,7 +29,17 @@ namespace GoogleCloudSamples
     // </summary>
     public class CommonTests
     {
-        private static readonly string s_projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        private static readonly string s_projectId = LoadProjectId();

```

Comment:
Aint there a fixture on these project that already loads the project ID? If there is, you should use it. If not, this is OK.

Reply:
There isn't a fixture in this file, and there's no dependency that pulls in a fixture either.